primitives/src/communities.rs

```diff
@@ -229,20 +230,46 @@ impl Location {
 	}
 }
 
-#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, TypeInfo)]
+#[derive(Encode, Decode, Clone, PartialEq, Eq, RuntimeDebug, PartialOrd, Ord, TypeInfo)]
 #[cfg_attr(feature = "serde_derive", derive(Serialize, Deserialize))]
 #[cfg_attr(feature = "serde_derive", serde(rename_all = "camelCase"))]
-pub struct UnboundedCommunityMetadata {
-	/// utf8 encoded name
-	pub name: UnboundedPalletString,
-	/// utf8 encoded abbreviation of the name
-	pub symbol: UnboundedPalletString,
-	/// IPFS cid to assets necessary for community branding
-	pub assets: IpfsCid,
-	/// ipfs cid for style resources
-	pub theme: Option<IpfsCid>,
-	/// optional link to a community site
-	pub url: Option<UnboundedPalletString>,
+pub enum AnnouncementSigner {
+	Substrate(MultiSigner),
+	Bip340([u8; 32]),
+}
+
+impl Default for AnnouncementSigner {
+	fn default() -> Self {
+		AnnouncementSigner::Bip340([0u8; 32])
+	}
+}
```

Comment:
Does it make sense to have a default here? Or is this because the storage is still a value query?

Reply:
I see it is only an option in the metadata. So I have to ask again. Why are we deriving a default?

Reply:
Not necessary, agreed